Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuth.h

```diff
@@ -140,6 +140,13 @@
 ///
 - (BOOL)clearStoredAccessTokens;
 
+///
+/// When NO, the Dropbox web login page will show a sign up link below the
```

Comment:
I see. This parameter was originally added for compliance in the other direction, since Apple was explicitly declining apps that _allowed_ signup...

Also, for reference, the `disable_signup` option also already removes the link to upgrade.

Anyway, as long as you leave the behavior the same for other SDK users that's fine.